### PR TITLE
OZ-658: Add config files and structure the project

### PR DESCRIPTION
**Pull request by VaishSiddharth**
- Add .gitignore
- Add parent and submodule with dependencies
- Add OpenMRS Watcher config
- Add config files (application.properties)
- Add folder structure (Handlers, Processors, Routes)

JIRA Link: https://mekomsolutions.atlassian.net/browse/OZ-658

### CODE REVIEW

File: openelis-openmrs/src/main/java/com/ozonehis/eip/openmrs/openelis/config/OpenmrsOpenelisConfig.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.openmrs.openelis.config;
```
Comment (rbuisson) [inline openelis-openmrs/src/main/java/com/ozonehis/eip/openmrs/openelis/config/OpenmrsOpenelisConfig.java]: Is this the package name?
The project is **eip-openelis-openmrs** (I don't know what is the agreed naming convention here)

Comment (VaishSiddharth) [reply openelis-openmrs/src/main/java/com/ozonehis/eip/openmrs/openelis/config/OpenmrsOpenelisConfig.java:8]: I think this should be fine.
WDYT @mks-d

Comment (VaishSiddharth) [reply openelis-openmrs/src/main/java/com/ozonehis/eip/openmrs/openelis/config/OpenmrsOpenelisConfig.java]: Let me refactor. Thanks for pointing out

**Review by mks-d — COMMENTED** [review]
* There is not ticket number in the PR title.
* There is no link to the ticket in the PR's first message/description.
* There is no description of the changes made in the PR's first message/description.

See https://www.notion.so/mekom/Code-of-Conduct-99fbb3a41ba44bb1947606a589aa35f9?pvs=4#bcf573f835a74651acf9a599355f2bab

**Pull request closed without merging**